imap_processing/tests/lo/test_lo_science.py

```diff
@@ -266,3 +297,30 @@ def test_validate_parse_events(sample_data, attr_mgr):
 
     assert dataset["de_count"].values == 1998
     assert dataset["passes"].values == 8
+
+
+def test_organize_spin_data(fake_spin_data):
+    # Arrange
```

Comment:
This got me to look up "Arrange, Act, Assert" - that's a cool framework/mnemonic.